parlai/core/torch_agent.py

```diff
@@ -505,7 +501,7 @@ def add_cmdline_args(cls, argparser):
             '--lr-scheduler',
             type=str,
             default='reduceonplateau',
-            choices=['reduceonplateau', 'none', 'fixed', 'invsqrt'],
+            choices=['reduceonplateau', 'none', 'fixed', 'invsqrt', 'cosine', 'linear'],
```

Comment:
since I moved the whole thing to the lr scheduler file, I feel like it's fine to keep it as is - keeping the choices in the classmethod for adding command lines feels more intuitive.

Reply:
Oh cool I didn’t quite notice. Sounds great.